test/kennel/models/slo_test.rb

```diff
@@ -44,6 +44,24 @@ def slo(options = {})
     it "stores options" do
       TestSlo.new(project, name: -> { "XXX" }).name.must_equal "XXX"
     end
+
+    describe "validates threshold" do
+      it "is ok when warning not set" do
+        Kennel::Models::Slo.new(project, thresholds: -> { [{ critical: 99 }] })
+      end
+
+      it "is invalid if warning > critical" do
+        assert_raises Kennel::Models::Record::ValidationError do
+          Kennel::Models::Slo.new(project, thresholds: -> { [{ warning: 0, critical: 99 }] })
+        end
+      end
+
+      it "is invalid warning != critical" do
```

Comment:
```suggestion
      it "is invalid warning == critical" do
```

Reply:
is that actually true ? / does the UI complain ?
... if not then we should not impose this, as `kennel` is meant to be a transparent layer over datadog and not opinionated

Reply:
![Screen Shot 2019-11-21 at 8 39 04 AM](https://user-images.githubusercontent.com/779148/69342852-6926ac00-0c3a-11ea-8d79-b7db0cab238f.png)